rebasehelper/cli.py

```diff
@@ -164,6 +178,9 @@ def run():
         except RebaseHelperError as e:
             logger.error('\n%s', six.text_type(e))
             sys.exit(1)
+        except EnableOptionFormatException as e:
+            sys.stderr.write("{}".format(e))
+            sys.exit(1)
```

Comment:
Well it's kinda is ... I wanted to stop using extra builder option in wrong format so I raised RebaseHelperError and I end up with no message at all. :D After some investigation I found that we are creating logger in Application.__init__.py so EVERY exception in CLI will not use logger cause it's created after argument parsing. 

I wanted to discuss with you this problem but you were not responding.